### DISCUSSION TITLE
Poll-Schumann model & emissions computation

### DISCUSSION THREAD

**Discussion by ChrisDeseure** [contrailcirrus/pycontrails#235] [Q&A] [answered]
Hi. My team and I have been using PS model and the associated emissions.py for a few months now. We have a few questions we'd like to discuss with the community. 
First of all we'd like to have more details about how take-off, initial climb and approach are managed. I see that the document “Poll & Schumann (2022). An estimation method for the fuel burn and other performance characteristics of civil transport aircraft. Part 3 Generalisation to cover climb, descent and holding.” is cited but I can’t find a way to easily get it. When we compute emissions on short flights we see that fuel estimates diverge from real flight data and I would be able to better understand why.

I've also seen that a default value for fuel deterioration factor has been implemented without clearly knowing :
o	the performance baseline used for the model (is PS fitting as much as possible the aircraft performance just out of factory?), 
o	if the default value that comes from the study of Gurrola Arrieta, M.D.J., Botez, R.M. and Lasne and only performed with a small regional jet (CRJ) is really applicable to the other aircraft types.

Thanks for your help!

**Comment by mlshapiro** [comment]
@roger-teoh do you mind to weigh in here?

**Comment by roger-teoh** [comment, marked as answer]
Hi Chris, 

Thank you for reaching out. 

With reference to the take-off, initial climb, and approach, the PS model only calculates the fuel consumption from the provided flight segments in the `Flight` object. In other words, if you obtain the flight trajectories from ADS-B telemetries, and the initial climb and final descent phase is not captured by ADS-B telemetry (which usually happens), then the PS model fuel consumption estimates would not capture or backfill these phases of flight. If the temporal resolution of the ADS-B telemetry is low (i.e., once a minute), then the flight trajectory might not adequately capture the final phase when the flight is at a holding pattern. The PS model also does not account for the fuel consumption burnt on the ground (take-off, landing, and taxi), and assumes a 70% load factor if the aircraft mass is not provided. Therefore, short-haul flights may have larger discrepancies relative to FDR data because: (i) the LTO phase covers a larger proportion of the overall fuel consumption; and (ii) the load factors might be higher than the assumed 70%. 

The PS model coefficients is fitted to data from the flight crew operating manual (FCOM) and assumes the performance of a newly produced aircraft. The default fuel deterioration factor of 2.5%, which assumes the mean deterioration in engine performance between maintenance cycle, comes from Gurrola et al and is supported by our comparison with FDR data from 7 different aircraft types. 

On the requested reference, i.e., Poll & Schumann (2022), it might be best to e-mail Professor Ian Poll to enquire about the status of this publication (d.i.a.poll@cranfield.ac.uk).

I hope this helps to clarify some of the PS model assumptions.
